aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -352,7 +352,10 @@ Tensor& empty_out(IntArrayRef size,
     return self.to(ScalarType::n, non_blocking);                 \
   }
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wmissing-prototypes"
```

Comment:
Could you leave a comment on why we do this here?

Reply:
OK